tests/test_message_bus.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from dbus_fast.errors import InvalidAddressError
+from dbus_fast.glib import MessageBus
+
+
```

Comment:
It looks like these tests just test that an exception was raised. I don't seen anything that tests that resources were not leaked.